src/components/Comments/comment.scss

```diff
@@ -0,0 +1,44 @@
+$primary-color: #00d0a0;
+#comment-button {
+  background-color: $primary-color !important;
+  color: white !important;
+  border-radius: 20px !important;
+  //padding: 5px !important;
```

Comment:
remove commented line

Reply:
Hello, Wayodi.
I am able to get my comments posted.
A small glitch though is that I have to reload the page after clicking on **Add Comment** button for the circle to be complete.

Regards
Steven Ennis

Reply:
> A small glitch though is that I have to reload the page after clicking on **Add Comment** button for the circle to be complete.

@Teatoller , I have madea few adjustments to tackle this. you can try again and see if it's okay.

Reply:
@jomasim I have made the changes. thanks for reviewing